android/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+# For more information about using CMake with Android Studio, read the
+# documentation: https://d.android.com/studio/projects/add-native-code.html
+
+# Sets the minimum version of CMake required to build the native library.
+
+cmake_minimum_required(VERSION 3.18.1)
+
+# Declares and names the project.
+
+project("bigwheels")
+
+add_subdirectory("../../../../.." "../../../../../app/build")
```

Comment:
So no, none of those variables exist. This CMakeLists file is the root CMakeLists.txt file that cmake invokes, so none of the normal BigWheels cmakes have run at this point.

I cleaned it up a bit to make it a little more explicit, but it's still ugly relative paths. Not sure what we can do about that. I'm open to suggestions!

Reply:
Do any/all of these relative path resolve to similar or the same locations? It might be easier to have a variable that starts a root and then appends the relative paths on. While `add_subdirectory` implies the current path - it's a little difficult to understand past like 3 relative paths.

Reply:
Yeah that's what my cleanup does.

However, Nathan is working on a subsequent PR that will completely clean up the directory structure and make all this relative path nonsense a thing of the past.